pkg/client/api.go

```diff
@@ -49,7 +49,7 @@ func (a *api) do(ctx context.Context, request interface{}, queryParams map[strin
 	if a.path == "" || a.method == "" || a.client == nil || a.jsonParser == nil {
 		panic("api not properly configured")
 	}
-
+	a.path = a.client.getHost() + "/" + consts.VmaasCmpAPIBasePath + "/" + a.path
```

Comment:
would recommend using fmt.Sprintf() here just to maintain consistency

Reply:
'+' have better perfomance then Sprintf, but changing to maintain consistency